docs/docs/gatsby-cli.md

```diff
@@ -38,13 +38,14 @@ Run `npx gatsby --help` for full help.
 
 `npx gatsby new gatsby-site`
```

Comment:
I'm not sure, but if we going to show code on new line maybe better to use code blocks to highlight the syntax:
```
> ```sh
> npx gatsby --help
> ```
```

Reply:
I like this approach 👍 especially when they are already on their own line. It might take more space than we actually want to.

Reply:
Generally I agree, but in this case I don't think anything will get highlighted? Let's merge this in and then re-visit once the page is live.